### PR TITLE
fix (content): #774 fix zoom, resize, high res issues

### LINKED ISSUE #774: Bug 1276932 - Infinite scrolling is not working after zooming out the Timeline page, if the scrollbar is not available
via https://bugzilla.mozilla.org/show_bug.cgi?id=1276932
<hr>
[Affected versions]:
- Firefox 45.0 and up
- Activity Stream 1.0.18

[Affected Platforms]:
- All Windows
- All Mac OS X
- All Linux

[Prerequisites]:
- Latest Activity Stream add-on version (v1.0.18) was previously installed on an old profile.
- Have enough History and Bookmarks in the profile so when scrolling is performed, a new batch of History items is listed.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites. 
2. Open Activity Stream Timeline from the browser toolbar.
3. Zoom out the Timeline page at lest 50%.
4. Refresh the Timeline page.

[Expected result]:
- Websites that were not displayed in viewport are listed while scrolling.

[Actual result]:
- Websites that were not displayed in viewport are not listed while scrolling because the scrollbar is not available.

[Notes]:
- This issue is reproducible when after zooming out the scroll bar is no longer displayed.
- On 4k monitors, this issue is reproducible instantly. When accessing Activity Stream Timeline there is no scroll bar and the user cannot load more visited websites while scrolling. 
- Attached a screen recording of the issue (https://goo.gl/01eYdy).

### PR DESCRIPTION
**Pull request by rlr**
r? @k88hudson what you think of this way to fix it?

Fixes #774 

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-stream/786)

### CODE REVIEW

File: content-src/components/TimelinePage/TimelineFeed.js [hunk 1]
```diff
@@ -56,6 +59,9 @@ const TimelineFeed = React.createClass({
     if (!prevProps.Feed.init && this.props.Feed.init) {
       this.refs.scrollElement.scrollTop = 1;
     }
+
+    // This is for the initial data load. Check if we can fit in more.
+    this.onScroll();
```
Comment (rlr) [inline content-src/components/TimelinePage/TimelineFeed.js]: Is there a better way to run this only after the initial data loads?

Comment (k88hudson) [inline content-src/components/TimelinePage/TimelineFeed.js]: but it in the block above

File: content-src/components/TimelinePage/TimelineFeed.js [hunk 2]
```diff
@@ -42,6 +42,9 @@ const TimelineFeed = React.createClass({
   },
   onResize: debounce(function() {
     this.windowHeight = window.innerHeight;
+
+    // After resizing, check if we can fit in more data.
+    this.onScroll();
```
Comment (k88hudson) [inline content-src/components/TimelinePage/TimelineFeed.js]: good idea 👍

**Comment by k88hudson** [comment]
Just move that one part into the `init` block and this is good to go 👍

File: content-src/components/TimelinePage/TimelineFeed.js (hunk 1 above)
Comment (rlr) [inline content-src/components/TimelinePage/TimelineFeed.js]: oooh, that's what that is for. Sweet

**Pull request merged**